article/app/views/liveblog/liveBlogBody.scala.html

```diff
@@ -72,10 +71,18 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(article.trail.h
 
                     @fragments.dropdown("Live feed", Some("live-feed"), true) {
                         <div class="js-article__container" data-component="body">
+                            @* Remove LiveFilter and the fragment once Toast comes out of AB test status *@
                             @fragments.liveFilter(article.fields.isLive)
-
-                            <div class="js-liveblog-body u-cf from-content-api js-blog-blocks @if(article.fields.isLive) {live-blog}" data-test-id="live-blog-blocks"
+                            <div class="js-liveblog-body u-cf from-content-api js-blog-blocks @if(article.fields.isLive) {live-blog}" data-most-recent-block="@article.mostRecentBlock" data-test-id="live-blog-blocks"
                                 itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
+                                <div class="toast__space-reserver">
+                                    <div id="toast__tofix" class="toast__container">
+                                        <a class="toast__button toast__button--closed button button--large">
```

Comment:
If this isn't a link (because it has no href), should it be a `button` instead? Unlike `a`s, `button`s are in the tab order so they're keyboard friendly :-)